### ISSUE TITLE
geoip2 dependency

### ISSUE THREAD

**Issue by micheloosterhof** [csirtgadgets/silver-meme#28]
When I do `pip install csirtgsdk` it pulls in `csirtg-indicator` (version 2.0), this in turn depends on `geoip2`. While `geoip2` is listed in `requirements.txt` it's not in the setup scripts, and therefor not automatically pulled in. Could you add `geoip2` as a dependency?

**Comment by wesyoung** [comment]
heya- we try to not pin too much in the setup.py as it tends to screw things too much in the other direction (ie: if we pinned the version in setup and you wanted a diff version in your app). have you tried pinning it in your own local app? (for BCP you should have your own requirements file where you can set the stable version that makes more sense for your app)

**Comment by micheloosterhof** [comment]
Hi! of course I can pin it in the local app, but the requirements in your current setup.py don't have versions pinned, and the ones in requirements.txt do. And all other requirements are listed in there, except geoip2.
I think i'm using basic functionality of this library and then I expect it to usually automatically handle the imports. 

I also see that you require 2.8-2.9 (from 2018) the current version of geoip2 is 4.1.0 (2020), do you have any reasons to think a newer version will not work?

michel .

**Comment by wesyoung** [comment]
i'll take a look- thinking back, i wanna say for some "heavier" deps (that require more system level libs) is why they were actually left out. that said- i thought we trapped those errors so things wouldn't fail if you didn't have them installed (v2 goes back a bit).. geoip has become a bit more stable (i think they were changing things a bit at the time too, it's fuzzy). i'm indifferent to adding a fuzzy pin  to setup.py if that makes things easier and you have a PR..

**Comment by micheloosterhof** [comment]
Yeah. A lot of people already do geoip enrichment in a central place (logstash, splunk), so it should probably not crash like it does not, but rather handle the missing library gracefully. 
People already opened up tickets in Cowrie last year because of this: https://github.com/cowrie/cowrie/issues/1468

**Comment by micheloosterhof** [comment]
Please keep in mind Cowrie is on a very old version of csirtg (0.0.0a22) and I'm trying to upgrade it to a more recent version, see also the other ticket I opened on the TOKEN variable.
